wordpress-mobile/WordPress-iOS
# Removing unused WPNoResultsView extensions

Ref #9504 

To test:

This just removes `WPNoResultsView+Model` and `WPNoResultsView+AnimatedBox`, which are no longer used. So... build and run! 😄
